### PR TITLE
RPC timeouts fixes

### PR DESCRIPTION
**Pull request by LukasGasior1**

### CODE REVIEW

File: src/main/scala/io/iohk/ethereum/jsonrpc/NetService.scala [hunk 1]
```diff
@@ -42,7 +42,7 @@ class NetService(nodeStatusHolder: Agent[NodeStatus], peerManager: ActorRef) {
 
   def peerCount(req: PeerCountRequest): ServiceResponse[PeerCountResponse] = {
     import akka.pattern.ask
-    implicit val timeout = Timeout(2.seconds)
+    implicit val timeout = Timeout(5.seconds)
```
Comment (adamsmo) [inline src/main/scala/io/iohk/ethereum/jsonrpc/NetService.scala]: should we add configuration for this?

Comment (mcsherrylabs) [inline src/main/scala/io/iohk/ethereum/jsonrpc/NetService.scala]: Because we've had to tune it then yes, yes we should.

**Comment by LukasGasior1** [comment]
@adamsmo @mcsherrylabs fixed

File: src/main/resources/application.conf [hunk 2]
```diff
@@ -152,6 +152,10 @@ grothendieck {
       # Enabled JSON-RPC APIs over the HTTP endpoint
       # Available choices are: eth, web3, net, personal
       apis = "eth,web3,net"
+
+      net {
+        peer-manager-timeout = 5.seconds
```
Comment (adamsmo) [inline src/main/resources/application.conf]: should we add this to `universal` and add description?

Comment (LukasGasior1) [inline src/main/resources/application.conf]: I wouldn't add it, it's internal thing

Comment (mcsherrylabs) [inline src/main/resources/application.conf]: (We're leaving out config that we don't really expect the user to change without having to talk to a developer. i.e. it's legit to leave it out of the universal file under certain conditions)

Comment (adamsmo) [inline src/main/resources/application.conf]: ok

**Pull request merged**